.gitignore

```diff
@@ -10,6 +10,8 @@
 /*.ipr
 /*.iws
 /*.iml
+/.project
+/.settings
```

Comment:
The .gitignore file should be the same as in the optaplanner repo:
  https://github.com/kiegroup/optaplanner/blob/master/.gitignore
there it does /.* to ignore those eclipse files